### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by Haidong-Kang** [GAIA-vision/GAIA-ssl#2]
good job.

I have some questions about Training details.
1. In your paper, “We train 200 epochs using a batch size of 1024 on 16 GPUs and an initial learning rate of 0.12”, which type of GPUs are used in your paper? V100?
2. you compute the Spearman [36] correlation between the similarity ranking and their final performance ranking. Why not use Kendall’s Tau correlation?
3. Is there any data on search costs?

thanks

**Comment by NickChang97** [comment]
1) Yes, we trained supernet with V100. 
2) We follow UnNAS, so we adapt the Spearman. And, Kendall's Tau is truly a more strict judgment.
3) I don't understand this question, can you explain it more ?
